### DISCUSSION TITLE
Foomuuri was the missing piece i was looking for to build a modern all-linux firewall

### DISCUSSION THREAD

**Discussion by lanefu** [FoobarOy/foomuuri#12] [Show and tell]
Thanks to finding Foomuuri I was able to replace my edgerouter-6 with a nanopi-r5s running Armbian.  

It's so easy to manage and works really well with ansible.  Thank you so much for creating this tool!

I've got a[ WIP foomuuri role in this ansible collection.](https://github.com/lanefu/ansible-collection-clammy/tree/main/roles/foomuuri)

I have an ansible repo showing a full router example implementation[ here ](https://github.com/lanefu/clammy-ng)

**Comment by kimheino** [comment]
> It's so easy to manage and works really well with ansible.

Great to hear! I don't use ansible myself but being easy to configure (=multiple configuration files) with such tools was one of the design goals.

> I've got a[ WIP foomuuri role in this ansible collection.](https://github.com/lanefu/ansible-collection-clammy/tree/main/roles/foomuuri)

Look good. I forwarded this link to one of my clients using ansible.

One suggestion: If you don't plan to change options in `foomuuri` section (for example `localhost_zone`) then it's better to just leave it out. All those options should have good default values. I tried to improve this in [wiki](https://github.com/FoobarOy/foomuuri/wiki/Configuration#foomuuri) too.

**Reply by lanefu** [reply]
Yeah I may switch to implicit config values and render them differently for `foomuuri.conf`  It's kind of an ansible pattern to define all the possible configuration values in a roles `defaults.yml`
